game/hiscore.cc

```diff
@@ -26,7 +24,7 @@ bool Hiscore::reachedHiscore(unsigned score, SongId songid, unsigned short level
 		if (elem.track != track) continue;
 		if (elem.level != level) continue;
 		if (score > elem.score) return true;
-		if (++position >= MaximumStoredScores) return false;
+		if (config["game/highscore_limit_stored_scores"].b() && ++position >= config["game/highscore_maximum_stored_scores"].ui()) return false;
```

Comment:
Some time ago we decided to write the if-/else-blocks into a new line. Don't know if we updated the wiki. But if you change a line, put the blocks into an own line please.